sys-apps/ignition/ignition-9999.ebuild

```diff
@@ -46,7 +46,7 @@ PATCHES=(
 
 src_compile() {
 	export GO15VENDOREXPERIMENT="1"
-	GO_LDFLAGS="-X github.com/flatcar-linux/ignition/v2/internal/version.Raw=${PV} -X github.com/flatcar-linux/ignition/v2/internal/distro.selinuxRelabel=false -X github.com/flatcar-linux/ignition/v2/internal/distro.writeAuthorizedKeysFragment=false" || die
+	GO_LDFLAGS="-X github.com/flatcar-linux/ignition/v2/internal/version.Raw=${PV} -X github.com/flatcar-linux/ignition/v2/internal/distro.writeAuthorizedKeysFragment=false" || die
```

Comment:
@tormath1 oh, I think writeAuthorizedKeysFragment should be true, right? Otherwise changes done by Ignition get lost when `update-ssh-keys` runs, or?

Reply:
and we would need to trigger `update-ssh-keys` once because `internal/authorized_keys_d/authorized_keys_d.go` is gone

Reply:
Here: https://github.com/flatcar-linux/Flatcar/issues/699